uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/sickness-leave-request-instance_test.js

```diff
@@ -114,5 +114,41 @@ define([
         expect(instance.required_documents).toEqual('');
       });
     });
+
+    describe('with required documents', function() {
+      beforeEach(function() {
+        instance = instance.init({}, false);
+      });
+
+      describe('for new leave request instance', function() {
+        it('is initialized to empty string', function() {
+          expect(instance.required_documents).toEqual('');
```

Comment:
You are already testing for this on line 114, *do not* copy paste stuff if you don't then double check it